### PR TITLE
Expand textview in the presence of a long text

### PR DESCRIPTION
**Pull request by rasmustautsglia**
This PR makes the text field expandable if the text is longer than there is room on a line. In total, the textfield can expand to up to 4 lines. If the text is longer than 4 lines, then the textfield becomes scrollable. 

In addition, the max character limit had been previously set to 200. I have increased it to 10000 characters to align it with web, which is also the API's limit.

MOB-2334

### CODE REVIEW

File: GliaWidgets/Sources/View/Chat/Entry/ChatMessageEntryView.swift [hunk 1]
```diff
@@ -219,17 +218,23 @@ class ChatMessageEntryView: BaseView {
...
-        if newHeight > kMaxTextViewHeight {
-            newHeight = kMaxTextViewHeight
-        } else if newHeight < kMinTextViewHeight {
-            newHeight = kMinTextViewHeight
+        textView.isScrollEnabled = Int(numLines) > maxTextLines
+
+        if Int(numLines) > maxTextLines {
+            if let lineHeight = textView.font?.lineHeight {
+                newHeight = lineHeight * CGFloat(maxTextLines)
+            }
```
Comment (yurii-glia) [inline GliaWidgets/Sources/View/Chat/Entry/ChatMessageEntryView.swift:236]:
```suggestion
        if Int(numLines) > maxTextLines {
            newHeight = font.lineHeight * CGFloat(maxTextLines)
```

File: GliaWidgets/Sources/View/Chat/Entry/ChatMessageEntryView.swift [hunk 2]
```diff
@@ -219,17 +217,23 @@ class ChatMessageEntryView: BaseView {
             height: CGFloat.greatestFiniteMagnitude
         )
 
-        // We need to take into account larger height
-        // of text view or placeholder label for
-        // updated text entry height.
-        var newHeight = max(textView.sizeThatFits(size).height, placeholderLabel.sizeThatFits(size).height)
+        guard let font = textView.font else {
```
Comment (yurii-glia) [inline GliaWidgets/Sources/View/Chat/Entry/ChatMessageEntryView.swift:220]: do we need to handle here not valid lineHeight that can lead to division by zero error?
```suggestion
        guard let font = textView.font, font.lineHeight > 0 else {
```

**Review by yurii-glia — APPROVED** [review]

**Review by EgorovEI — APPROVED** [review]

File: GliaWidgets/Sources/View/Chat/Entry/ChatMessageEntryView.swift (hunk 1 above)
Comment (rasmustautsglia) [reply GliaWidgets/Sources/View/Chat/Entry/ChatMessageEntryView.swift]: Fixed [04c7b69](https://github.com/salemove/ios-sdk-widgets/pull/661/commits/04c7b6911627931a298245bae76d014f53953098)

**Comment by rasmustautsglia** [comment]
!squash

**Pull request merged**